spec/views/application/_navbar.html.erb_spec.rb

```diff
@@ -2,27 +2,43 @@
 
 require 'rails_helper'
 RSpec.describe 'application/_navbar.html.erb', type: :view do
-  let(:current_user) { FactoryBot.create :user }
+  context 'the current user is not an admin' do
+    let(:current_user) { FactoryBot.create :user }
 
-  before do
-    sign_in current_user
-    allow(view).to receive(:current_user).and_return(current_user)
-    render
-  end
+    before do
```

Comment:
Same code as below, pull out of the context and put it straight under the describe